src/App.js

```diff
@@ -1,29 +1,27 @@
 import React from 'react';
+import { PostList } from './components/PostList/PostList';
 
 import './App.scss';
 
 import posts from './api/posts';
-import comments from './api/comments';
+import comms from './api/comments';
 import users from './api/users';
 
+const preparedPosts = posts.map((post) => {
+  const user = users.find(person => person.id === post.userId);
+  const comments = comms.filter(commnt => post.id === commnt.postId);
```

Comment:
можно не экономить буквы)
```suggestion
  const filteredComments = comments.filter(comment => post.id === comment.postId);
```